SeaBattle/Point.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace SeaBattle
+{
+    public struct PointStruct
```

Comment:
There is a Point struct in .net, namespace is System.Drawing

Reply:
I can use System.Drawing instead of this class then?

Reply:
I think so, why not?